rpgJavaInterpreter-core/src/main/kotlin/com/smeup/rpgparser/interpreter/ISymbolTable.kt

```diff
@@ -0,0 +1,45 @@
+package com.smeup.rpgparser.interpreter
+
+interface ISymbolTable {
```

Comment:
I think that adding an I in front of an interface couples too much to a particular design choice. What if we want to use an abstract class later? Obviously this is just a matter of taste, so every choice is a good one :-)

Reply:
We could talk about it in the next future, but I agree that my choice could be not the best choice :-)